class ExceptionClienteYaExiste(Exception):
    def __init__(self):
        pass


class ExceptionEmpresaYaExiste(Exception):
    def __init__(self):
        pass